test/Recipe-test.js

```diff
@@ -27,11 +27,12 @@ describe('Recipe', () => {
     expect(recipe.name).to.equal("Loaded Chocolate Chip Pudding Cookie Cups")
     expect(recipe.instructions).to.deep.equal(recipeSample[0].instructions)
     expect(recipe.tags).to.deep.equal(recipeSample[0].tags);
+    expect(recipe.ingredientsData).to.deep.equal(ingredientSample)
```

Comment:
I realized that we will also have to add in an assertion for the ingredients property :
`expect(recipe.ingredients).to.deep.equal(recipeSample[0].ingredients)`